src/main.py

```diff
@@ -47,18 +79,20 @@ def force_shutdown(self, timeout: float = 1.0):
         """
         self.sender.force_shutdown(timeout)
 
-    def send_event(self, event_category: str, event_action: str, event_label: str, event_value: int = 1, **kwargs):
+    def send_event(self, event_category: str, event_action: str, event_label: str, event_value: int = 1,
+                   force_send=False, **kwargs):
         """
         Send single event.
 
         :param event_category: category of the event
         :param event_action: action of the event
         :param event_label: the label associated with the action
         :param event_value: the integer value corresponding to this label
+        :param force_send: forces to send event ignoring the consent value
         :param kwargs: additional parameters
         :return: None
         """
-        if self.consent:
+        if self.consent or force_send:
```

Comment:
Yes, we explicitly clarified this with Leandro and legal. We are allowed to send opt-in result information, the only requirement is not to store user ID on the system. So in case if user didn't consent for sending telemetry, random value is used as ID and not stored on the system.